src/main/java/com/stuypulse/robot/commands/DrivetrainAutomaticAlign.java

```diff
@@ -5,24 +5,32 @@
 
 public class DrivetrainAutomaticAlign extends DrivetrainGoalCommand {
 
+    private static class ShooterAligner extends DrivetrainGoalAligner {
+
+        // holds the shooter because the shooter mode / distance changes
+        private final Shooter shooter;
+
+        public ShooterAligner(Shooter shooter) {
+            super(0); // default distance of zero
+            this.shooter = shooter;
+        }
+
+        private Number getUpdatedDistance() {
```

Comment:
This function seems really redundant as its only called once.

I understand that it helps with readability, but its confusing so see this function defined here as it leads me to expect it to be used in multiple spots.